src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -49,32 +49,15 @@ export default class AuthorizationService {
   }
 
   public static async checkAndGetSiteAuthorizationFilters(tenant: Tenant, userToken: UserToken,
-      filteredRequest: HttpSiteRequest): Promise<AuthorizationFilter> {
+      filteredRequest: Record<string, any>, authAction: Action): Promise<AuthorizationFilter> {
```

Comment:
The type change here looks wrong. Why have you make the `filteredRequest` type a lot less strict?